gradle-plugin/src/main/kotlin/xyz/block/gradle/Flags.kt

```diff
@@ -58,7 +60,7 @@ val Project.sandbagVersion: String
     val result = sandbagHashFile.useLines { lines ->
       return@useLines lines.firstOrNull { line ->
         return@firstOrNull line.substringBefore('|') == project.path
-      } ?: throw GradleException("$path not found in hashing file. Please re-run sandbag tool.")
```

Comment:
I removed this because the settings plugin applies the publish plugin to all projects in the build, which include the "empty" intermediate projects (for example `:apps`). These projects have no sources (at least in our build), apply no plugins that produce a publishable component, and are never in the hashing output file. 

We avoided this previously by baking the publishing into other base conventions plugins which were individually applied at the leaf node projects.

as an aside, I think we could revisit this entire hashing file input. The plugin could hash the files itself to determine the version, this mechanism is kinda awkward